github/users_administration.go

```diff
@@ -38,13 +38,19 @@ func (s *UsersService) DemoteSiteAdmin(ctx context.Context, user string) (*Respo
 	return s.client.Do(ctx, req, nil)
 }
 
+// userSuspendOptions represents the reason a user is being suspended.
+type userSuspendOptions struct {
+	Reason string `json:"reason,omitempty"`
```

Comment:
Let's change this from `string` to `*string` so that the `omitempty` can do its job and not pass it if it is not provided.